js/marquee.js

```diff
@@ -0,0 +1,48 @@
+class Marquee {
+    constructor(element) {
+      this.container = element;
+      this.data;
+      this.container;
+    }
+  
+    start() {
+      this.MarqueeData();
+    }
+  
+    async MarqueeData() {
+      const url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/quotes/`;
+      const data = await getJsonFromServer(url);
+      if (data) {
+        this.printMarquee(data);
+      }
+      
+      return console.log('Error');
+    }
+  
+    printMarquee(data) {
+      const containerMarq = makeElement('div');
+  
+      for (const element of data) {
+        const symbolMarquee = makeElement('p', {
+          className: 'symbol-marquee',
+          innerHTML: `${element.symbol} &nbsp;`,
+        });
+        const sign = getCssSign(element.changesPercentage);
+        const symbolPrice = makeElement('span', {
+          className: `symbol-price ${sign}`,
+          innerText: `${element.price}`,
+        });
+  
+        symbolMarquee.append(symbolPrice);
```

Comment:
Although you code works, DOM manipulation is an action that requires a lot of computing power (DOM access is "expensive" for the browser) and since then it should be always done outside of loop iteration. A better approach would be to aggregate all the new elements inside a parent element that does not live inside the DOM yet, and only after the loop iteration is done you should append the parent element with all the children inside of it. You can also check about createDocumentFragment() (google it!)